### ISSUE TITLE
On 2nd monitor, the dock should hide or push back the windows in overview.

### ISSUE THREAD

**Issue by ghost** [micheleg/dash-to-dock#785]
[Here's the image.](https://i.imgur.com/nVTrG8a.png)

**Comment by micheleg** [comment]
Yes,  I have recently noticed this annoyng bug on my work computer after upgrading to Ubuntu 18.04 (GS 2.28), but it seems to be working fine on older versions (e.g. Debian Stable, GS 3.22). I also think it works fine if the dock iis on the left. Might be a trivial fix but need to dig into the code.

**Comment by Ufeldraku** [comment]
Oh nice.

**Comment by eantropix** [comment]
Image seems to have been deleted. Would be nice to have it as reference. Maybe update the link?
